### PR TITLE
Ftr/default language

### PR DESCRIPTION
**Pull request by ineelhere**
### Changes
* imported `useEffect` from react
* used `useEffect` hook to trigger the `i18n.changeLanguage('en')` call when the component mounts

### How to test
* pull this branch
* package and run the app on local
* All text should appear in English by default
* user should be also able to switch to Francias

### CODE REVIEW

**Review by kamilzyla — CHANGES_REQUESTED** [review]

**Review by kamilzyla — CHANGES_REQUESTED** [review]
Are we sure of this change? French was made the default on an explicit request from Robbie (#199). If we do want to proceed, see my comment in the code :slightly_smiling_face:

File: src/renderer/containers/App.tsx [hunk 1]
```diff
@@ -16,6 +16,10 @@ export default function App(props: Props) {
   const { t, i18n } = useTranslation();
   useAnalytics();
 
+  useEffect(() => {
+    i18n.changeLanguage('en');
+  }, [i18n]);
+
```
[the remark this answers is not in the document]
Comment (ineelhere) [reply src/renderer/containers/App.tsx]: Wow! This is so cool 😮 In my quest to learn and apply TypeScript, I over-engineered things 😅 
I've updated the PR with the change 🙂

**Comment by ineelhere** [comment]
> Are we sure of this change? French was made the default on an explicit request from Robbie (#199). If we do want to proceed, see my comment in the code 🙂

@magdakunat @Marundu could you please join @kamilzyla in this thread regarding if we are sure of this change? 🙂 
 
In case we do want to proceed, I have already pushed the changes you've suggested in the review, @kamilzyla 🙂

**Review by kamilzyla — APPROVED** [review]
You have a green light from me on the technical side; feel free to merge once we confirm that we want this change :+1:

**Comment by Marundu** [comment]
Mbaza is being adopted/ will be adopted by a wider number of users. It makes sense to have the default language as Englihs, since that is more widely spoken than French. This is why I requested for the change.

**Comment by magdakunat** [comment]
> > Are we sure of this change? French was made the default on an explicit request from Robbie (#199). If we do want to proceed, see my comment in the code 🙂
> 
> @magdakunat @Marundu could you please join @kamilzyla in this thread regarding if we are sure of this change? 🙂
> 
> In case we do want to proceed, I have already pushed the changes you've suggested in the review, @kamilzyla 🙂

This is @Marundu and Andrzej Białaś decision as the D4G leaders

**Pull request merged**